pkg/reader/usdc_reader.go

```diff
@@ -20,6 +18,9 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
 )
 
+// USDCMessageReader retrieves each of the CCTPv1 MessageSent event created
+// during the Commit phase of the USDC token transfer. The events are created
```

Comment:
nit, but `MessageSent` events are created when a message is sent via Router - as a part of that transaction OnRamp interacts with USDCTokenPool, which interacts with Circle contracts. Technically, it's before Commit (assuming Sol works similar to EVM)